extras/indexing/src/test/java/org/apache/rya/indexing/entity/query/EntityQueryNodeTest.java

```diff
@@ -152,7 +153,7 @@ public void constructor_wrongType() throws Exception {
 
     @Test
     public void evaluate_constantSubject() throws Exception {
-        final MongoClient client = MockMongoFactory.newFactory().newMongoClient();
+        final MongoClient client = MockMongoSingleton.getInstance();
```

Comment:
I think there could be value to accessing the MongoClient from MongoITBase (instead of the MockMongoSingleton.getInstance()), or that `@Rule` I mentioned.